### PR TITLE
fix(ui5-menu): provide accessible name to the popover

### PR DESCRIPTION
**Pull request by unazko**
Issue:
- Every popover should have an accessible name
according to the accessibility standard.

Fixes: #9313

### CODE REVIEW

**Review by NHristov-sap — APPROVED** [review]

File: packages/main/src/Menu.ts [hunk 1]
```diff
@@ -269,6 +270,10 @@ class Menu extends UI5Element {
 		return this.items.filter((item): item is MenuItem => !item.isSeparator);
 	}
 
+	get menuAccessibleName() {
```
Comment (tsanislavgatev) [inline packages/main/src/Menu.ts:273]: do we have any other accessible name? As we call the getter "menuAccessibleName" when we are in the menu file. I would suggest something like "acessibleNameText"

Comment (unazko) [reply packages/main/src/Menu.ts]: done

File: packages/main/src/i18n/messagebundle.properties [hunk 2]
```diff
@@ -508,6 +508,9 @@ MENU_BACK_BUTTON_ARIA_LABEL=Back
 #XACT: ARIA announcement for the Menu Close button aria-label attribute
 MENU_CLOSE_BUTTON_ARIA_LABEL=Decline
 
+#XACT: Aria information for the Menu popover
```
Comment (BorisDafov) [inline packages/main/src/i18n/messagebundle.properties:511]: correct acronym is:
ARIA

**Review by BorisDafov — CHANGES_REQUESTED** [review]
minor

**Review by BorisDafov — COMMENTED** [review]
I approve the PR from UA perspective

**Review by tsanislavgatev — APPROVED** [review]

**Pull request merged**